ranger-core/src/test/java/io/appform/ranger/core/serviceprovider/ServiceProviderTest.java

```diff
@@ -98,29 +98,37 @@ protected NodeDataSink<T, TestSerializer<T>> dataSink(Service service) {
         }
     }
 
-    @Test(expected = NullPointerException.class)
-    public void testInvalidServiceProvider(){
-        new TestServiceProviderBuilder<>()
-                .withServiceName("test-service")
-                .withNamespace("test")
-                .withHostname("localhost-1")
-                .withPort(9000)
-                .build();
+    @Test
+    void testInvalidServiceProvider() {
+        try {
+            new TestServiceProviderBuilder<>()
+                    .withServiceName("test-service")
+                    .withNamespace("test")
+                    .withHostname("localhost-1")
+                    .withPort(9000)
+                    .build();
+        } catch (NullPointerException exception) {
```

Comment:
You can use `Assertions.assertThrowsExactly(NullPointerException.class, () -> {});`

Reply:
Fix in all places